TESTS/psa/entropy_inject/main.cpp

```diff
@@ -16,7 +16,7 @@
 * limitations under the License.
 */
 
-#if ((!defined(TARGET_PSA) || (!defined(COMPONENT_PSA_SRV_IPC)) && !defined(MBEDTLS_ENTROPY_NV_SEED)))
+#if (!defined(TARGET_PSA) || ((!defined(COMPONENT_PSA_SRV_IPC)) && !defined(MBEDTLS_ENTROPY_NV_SEED)))
```

Comment:
please double check this expression - looks suspicious to me
too many parenthesis here